modules/py-environment/modulefile.template

```diff
@@ -22,7 +22,7 @@ if {[info exists env(PYTHONPATH)] && ${esc}env(PYTHONPATH) != ""} {
 }
 
 # This disables loading modules and .pth files from ~/.local/lib/python2.7/...
```

Comment:
It's worth updating this comment too

Reply:
I thought that comment was still correct; how would you want it updated?

Reply:
It would be more useful to say why it's being set. The old one didn't do that very well either, only explaining what the `PYTHONNOUSERSITE` variable was for.

Perhaps "# Make user packages folder unique to each module to avoid dependency conflicts", or similar?